src/mysky/index.ts

```diff
@@ -558,4 +667,8 @@ export class MySky {
   protected async signRegistryEntry(entry: RegistryEntry, path: string): Promise<Signature> {
     return await this.connector.connection.remoteHandle().call("signRegistryEntry", entry, path);
   }
+
+  protected async signEncryptedRegistryEntry(entry: RegistryEntry, path: string): Promise<Signature> {
+    return await this.connector.connection.remoteHandle().call("signEncryptedRegistryEntry", entry, path);
+  }
```

Comment:
do not await returned value, it's already a promise

```suggestion
  protected signEncryptedRegistryEntry(entry: RegistryEntry, path: string): Promise<Signature> {
    return this.connector.connection.remoteHandle().call("signEncryptedRegistryEntry", entry, path);
  }
```

Reply:
https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise

Reply:
The link you pasted enforces the request to change it, I'm not sure why you gave it thumbs down?

Reply:
https://eslint.org/docs/rules/no-return-await